main.go

```diff
@@ -134,11 +136,15 @@ func loadOrgs(o options) (map[string]org.Config, error) {
 			admins := getGlobalTeam(cfg, Admins)
 			maintainers := getGlobalTeam(cfg, Maintainers)
 			approvers := getGlobalTeam(cfg, Approvers)
+			tsc := getGlobalTeam(cfg, TSC)
+			gb := getGlobalTeam(cfg, GB)
 
 			for name := range cfg.Repos {
 				admins.Repos[name] = github.Admin
 				maintainers.Repos[name] = github.Maintain
 				approvers.Repos[name] = github.Write
+				tsc.Repos[name] = github.Write
+				gb.Repos[name] = github.Write
```

Comment:
@beeme1mr do you think write permissions are enough, or should we elevate the permissions to admin or maintain?